app/services/proofing/mock/ddp_mock_client.rb

```diff
@@ -32,12 +32,33 @@ class DdpMockClient < Proofing::Base
 
       proof do |applicant, result|
         result.transaction_id = TRANSACTION_ID
+
         response_body = File.read(
           Rails.root.join(
             'spec', 'fixtures', 'proofing', 'lexis_nexis', 'ddp', 'successful_response.json'
           ),
         )
-        result.review_status = case SsnFormatter.format(applicant[:ssn])
+        result.review_status = review_status(
+          session_id: applicant[:threatmetrix_session_id],
+          ssn: applicant[:ssn],
+        )
+
+        result.response_body = JSON.parse(
+          response_body.gsub('REVIEW_STATUS', result.review_status.to_s),
```

Comment:
I think this is not wired up correctly--I'm not seeing `REVIEW_STATUS` in that file. In an case, can we do this without a `gsub` (parse the JSON, modify, then re-serialize?)

Reply:
updated in cbb1e17d0522744fec6a9c3b2737c75577014390